book/src/template_syntax.md

```diff
@@ -658,6 +658,16 @@ E.g. to test if the least significant bit is set in an integer field:
 {% endif %}
 ```
 
+### Type conversion
+
+You can use the operator [`as`](https://doc.rust-lang.org/std/keyword.as.html) in `{{ … }}`
+expressions, and `{% … %}` blocks. It works the same as in Rust, but with some deliberate
+restrictions:
+
+- You can only use [primitive types](https://doc.rust-lang.org/std/primitive/index.html)
+  like `i32` or `f64` both as source variable type and as target type.
+- If the source is a reference to a primitive type, e.g. `&&&bool`, then rinja automatically
+  dereferences the value until it gets get the underlying `bool`.
```

Comment:
```suggestion
- If the source is a reference to a primitive type, e.g. `&&&bool`, then `rinja` automatically
  dereferences the value until it gets the underlying `bool`.
```

Some parts of the suggestion were skipped. ;)

Reply:
Oops, fixed now. Thank you for reading my texts, twice. :D